kore/src/Kore/Internal/TermLike.hs

```diff
@@ -578,8 +578,21 @@ instance
     (SortedVariable variable, Unparse variable) =>
```

Comment:
The `Unparse variable` constraint is redundant now because of using `toVariable` below.